sketch.js

```diff
@@ -1,40 +1,40 @@
-function setup() {
-  createCanvas(600,600);
-
-}
-
-function draw() {
-  background (200,50,0);
+  function setup() {
+    createCanvas(600,600);
   
-  noStroke()
-  fill(300,100,0)
-  rect(0,100,600,100)
+  }
   
-  noStroke()
-  fill(300,200,0)
-  rect(0,200,600,100)
+  function draw() {
+    background (200,50,0);
+    
+    noStroke()
+    fill(300,100,0)
+    rect(0,100,600,100)
+    
+    noStroke()
```

Comment:
Did you try omitting `noStroke()` for the rectangles in a row where it doesn't change? 
Code is often clearer when there is left of it. 

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself